curation-migration-datasync/types.ts

```diff
@@ -12,11 +12,28 @@ export type ScheduledItemPayload = {
   isSyndicated: boolean;
   createdAt: number;
   createdBy: string;
+  updatedBy?: string;
   updatedAt: number;
   scheduledSurfaceGuid: string;
   scheduledDate: string;
 };
 
+export type ApprovedItemPayload = {
+  eventType: string;
+  approvedItemExternalId: string;
+  url: string;
+  title?: string | null;
```

Comment:
kept them as optional , so if the value is `undefined` or `null`, we will get old value set back to them